api/tests/Api/ActivityProgressLabel/CreateActivityProgressLabelTest.php

```diff
@@ -0,0 +1,129 @@
+
+    public function testCreateActivityProgressLabelIsNotPossibleForInactiveCollaboratorBecauseCampIsNotReadable() {
+        static::createClientWithCredentials(['email' => static::$fixtures['user5inactive']->getEmail()])
+            ->request('POST', '/activity_progress_labels', ['json' => $this->getExampleWritePayload()])
+        ;
+        $this->assertResponseStatusCodeSame(400);
+        $this->assertJsonContains([
+            'title' => 'An error occurred',
+            'detail' => 'Item not found for "'.$this->getIriFor('camp1').'".',
+        ]);
+    }
+
+    public function testCreateActivityProgressLabelIsDeniedForGuest() {
+        static::createClientWithCredentials(['email' => static::$fixtures['user3guest']->getEmail()])
+            ->request('POST', '/activity_progress_labels', ['json' => $this->getExampleWritePayload()])
+        ;
+
+        $this->assertResponseStatusCodeSame(403);
+        $this->assertJsonContains([
+            'title' => 'An error occurred',
+            'detail' => 'Access Denied.',
+        ]);
+    }
+
+    public function testCreateActivityProgressLabelIsAllowedForMember() {
+        static::createClientWithCredentials(['email' => static::$fixtures['user2member']->getEmail()])
+            ->request('POST', '/activity_progress_labels', ['json' => $this->getExampleWritePayload()])
+        ;
+
+        $this->assertResponseStatusCodeSame(201);
+        $this->assertJsonContains($this->getExampleReadPayload());
+    }
+
+    public function testCreateActivityProgressLabelIsAllowedForManager() {
+        static::createClientWithCredentials()
+            ->request('POST', '/activity_progress_labels', ['json' => $this->getExampleWritePayload()])
+        ;
+
+        $this->assertResponseStatusCodeSame(201);
+        $this->assertJsonContains($this->getExampleReadPayload());
+    }
+
+    public function testCreateActivityProgressLabelValidatesMissingCamp() {
+        static::createClientWithCredentials()
+            ->request('POST', '/activity_progress_labels', ['json' => $this->getExampleWritePayload([], ['camp'])])
+        ;
+
+        $this->assertResponseStatusCodeSame(422);
+        $this->assertJsonContains([
+            'violations' => [
+                [
+                    'propertyPath' => 'camp',
+                    'message' => 'This value should not be null.',
+                ],
+            ],
+        ]);
+    }
+
+    public function testCreateActivityProgressLabelValidatesMissingTitle() {
+        static::createClientWithCredentials()
+            ->request('POST', '/activity_progress_labels', ['json' => $this->getExampleWritePayload([], ['title'])])
+        ;
+
+        $this->assertResponseStatusCodeSame(422);
+        $this->assertJsonContains([
+            'violations' => [
+                [
+                    'propertyPath' => 'title',
+                    'message' => 'This value should not be blank.',
+                ],
+            ],
+        ]);
+    }
+
```

Comment:
suggestion: either use Gedmo\Sortable or verify the unique constraint on position

Optionally:
\+ test trimming of title
\+ test length of title is validated